tools/src/format.rs

```diff
@@ -102,7 +102,12 @@ pub fn format_code(jumps: &HashMap<String, Jump>, format: FormatAs,
 
         let data = match (&token.kind, datum) {
             (&tokenize::TokenKind::Identifier(None), Some(d)) => {
-                let ref id = d[0].sym;
+                let id = if d.len() == 1 {
```

Comment:
Maybe add a comment of what you're trying to do? It's not _that_ obvious without reading the commit message. But you can figure out looking a bit more carefully at the code, so whatever you prefer really.